projects/components/src/public-api.ts

```diff
@@ -296,6 +296,16 @@ export * from './select/select-control-option.component';
 export * from './select/select.component';
 export * from './select/select.module';
 
+// Splitter
+export * from './splitter/splitter';
+export * from './splitter/splitter-content.directive';
+export * from './splitter/splitter.component';
+export * from './splitter/splitter.module';
+
+// Multi-select
+export * from './multi-select/multi-select.component';
+export * from './multi-select/multi-select.module';
```

Comment:
why this is needed, I guess it's not.

Reply:
it should be exported already. Let me remove it.